### PR TITLE
Update PAO dependency

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

File: .githooks/components.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ components=( \
 	["features"]="feature-configs" \
 	["ztp"]="ztp" \
 	["infra"]="Makefile",".githooks/","hack/","openshift-ci/" \
-	["vendor"]="vendor","cnf-tests","ztp", "go.mod","go.sum" \
+	["vendor"]="vendor","cnf-tests","ztp","go.mod","go.sum" \
```
Comment (yuvalk) [inline .githooks/components.sh:10]: sorry!

Comment (yuvalk) [reply .githooks/components.sh:10]: tnx for fixing this

Comment (cynepco3hahue) [reply .githooks/components.sh:10]: no worries :)

**Pull request merged**